tests/sqllab_tests.py

```diff
@@ -0,0 +1,167 @@
+"""Unit tests for Sql Lab"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import csv
+import json
+import io
+import unittest
+
+from flask_appbuilder.security.sqla import models as ab_models
+from caravel import db, models, utils, appbuilder, sm
+from .base_tests import CaravelTestCase
+
+
+class SqlLabTests(CaravelTestCase):
+    """Testings for Sql Lab"""
+
+    def __init__(self, *args, **kwargs):
+        super(SqlLabTests, self).__init__(*args, **kwargs)
+
+    def setUp(self):
+        db.session.query(models.Query).delete()
+        self.run_sql("SELECT * FROM ab_user", 'admin', client_id='client_id_1')
+        self.run_sql("SELECT * FROM NO_TABLE", 'admin', client_id='client_id_3')
+        self.run_sql("SELECT * FROM ab_permission", 'gamma', client_id='client_id_2')
+
+    def tearDown(self):
+        pass
+
+    def test_sql_json(self):
+        data = self.run_sql('SELECT * FROM ab_user', 'admin', "1")
+        assert len(data['data']) > 0
```

Comment:
while you are here, could you please refactor asserts into assertEquals, assertLess, etc ?